pkg/scheduler/framework/plugins/dynamicresources/dynamicresources.go

```diff
@@ -237,12 +250,35 @@ var _ framework.PreScorePlugin = &dynamicResources{}
 var _ framework.ReservePlugin = &dynamicResources{}
 var _ framework.EnqueueExtensions = &dynamicResources{}
 var _ framework.PostBindPlugin = &dynamicResources{}
+var _ framework.RequeuePlugin = &dynamicResources{}
 
 // Name returns name of the plugin. It is used in logs, etc.
 func (pl *dynamicResources) Name() string {
 	return Name
 }
 
+func (pl *dynamicResources) Requeue(ctx context.Context, p *v1.Pod, event *framework.ClusterEvent, oldObj, obj interface{}) framework.SchedulingHint {
+	if !event.In(pl.EventsToRegister()) {
+		// not interested event.
+		return framework.PodNotAffected
+	}
+
+	switch event.Resource {
+	case framework.ResourceClaim:
+		return pl.isSchedulableAfterClaimChange(oldObj, obj, p)
+	case framework.PodSchedulingContext:
+		return pl.isSchedulableAfterPodSchedulingChange(oldObj, obj, p)
+	case framework.Node:
+		// TODO(pohly): check this event in detail to determine whether this event really make Pod schedulable.
+		return framework.PodMaybeSchedulable
+	case framework.WildCard:
```

Comment:
Can we get here, i.e. is `event.In` returning true for `framework.WildCard`?

Reply:
Yes.
https://github.com/kubernetes/kubernetes/blob/25f25a1d1b28bf10757b2c5b941e3bd4e0889104/pkg/scheduler/framework/types.go#L93

Reply:
On the second thought, it's probably better to just return `true` for `framework.WildCard` event in `isPodWorthRequeuing` instead of executing Requeue()s. because framework.WildCard is kind of special event and we only use it for periodically flush of unschedulable Pod pool.
https://github.com/kubernetes/kubernetes/blob/master/pkg/scheduler/internal/queue/events.go#L86